src/update.js

```diff
@@ -145,10 +147,8 @@ module.exports = async (octokit, context) => {
                 opts.name = tag_name
             }
 
-            if (!body) {
-                opts.body = `Release based on tag **${tag_name}**. Enjoy! 🎉`
-            }
-
+            // When creating we default to initialBody if available, otherwise body, or our fallback value.
+            opts.body = initialBody || body || `Release based on tag **${tag_name}**. Enjoy! 🎉`
```

Comment:
@brunchboy, looking at this code i think the `initialBody` variable achieves the same as `body` variable.

Makes me wonder what's exactly the use case?... is it that in case you don't know if the release exists you want  your release description with a value different to when it's an edit, right?. If that's the case, can a mode that checks if a release already exists would be more helpful? so you can use an if value on your next step.

What do you think? 🤔

Reply:
I tried to explain the use case in the documentation. I need to provide a body when creating a release but never change the body if the release exists. I can do that by providing `initialBody` and no `body`. I don’t want to have multiple steps relating to releases.

If this is a direction that differs from what you want to support, that’s ok, I will just use my fork. But it would  be nice to be able to stay on you release train.

Reply:
An alternate direction that could work for me, if you prefer it, would be to stick with just one body markdown value, but add another boolean flag which specifies whether to update the body on existing releases.

Reply:
Oh ok, i just missunderstood the use case... That's something i also wanted i was pretty sure the logic was there already, since it makes sense that if you want to edit the release you might want to edit only the values that you set. That's why the [if no `body` condition is only present when the release was not found](https://github.com/Xotl/cool-github-releases/blob/master/src/update.js#L148).

Did it not preserved the original values during edit?. 🤔

Reply:
I am sorry I am not being clear, it seems.

It is true that if I leave out the `body_mrkdown:` entry in my action configuration, it leaves the body of my release alone. That works for cases where the release exists, has a description, and I may have manually updated it. But that does not work when I am creating a new release, I do not want the default hardcoded text to be used, I want to supply my own custom text that should be used when first creating the release. My workflow doesn’t know (and I don’t want it to have to know) if this is the first, second, or hundredth time a snapshot release is being built. Here is an example of how I am using the new option in my fork, which may make it more clear: https://github.com/Deep-Symmetry/carabiner/blob/master/.github/workflows/build.yml#L46-L61

That way, my custom text gets installed the first time a release is created, but I can go and manually edit it to add more information to my users if I want, and know that my manual edits will be preserved even if this workflow runs and uploads updated artifacts to the release.

Reply:
Maybe i'm overlooking something but shouldn't you get the exact same behavior if you change `initial_mrkdwn` to `body_mrkdwn`?. The first time you create and the _Nth_ time you edit a release you will get the exact same description, using `initial_mrkdwn` or `body_mrkdwn` on your current forked version.

Let's say that you set `body_mrkdwn`, since `initial_mrkdwn` it's not set i will use the `body_mrkdwn` value which is set, so while createing a new release it will have your custom value as expected. During an edit you can set the `body_mrkdwn` again and will have the exact set value as expected.

Or am i overlooking something?. Probably don't understand correctly the use case yet.

Reply:
No, if I use `initial_mrkdwn`, on the _Nth_ release the description does not change, and the manual edits I made to the release description after the release was created using my web browser on GitHub are preserved.

If I use `body_mrkdwn`, the action replaces the description with what is configured in the action, and all manual edits I made to the release description using my web browser on GitHub are lost.

Reply:
Also, i'm able to set an initial value while creating new releases with the current state of the action and since 1.0.7.

Reply:
oh, i think i finally got it. 😅

Reply:
Great! 😄 I understand that this is a strange workflow for people who are not accustomed to working with Maven snapshot releases, which is why I am happy to move off onto my own fork if it is too different from what you imagined, but they are very common in the Java world so I am probably not the only person who would find this feature useful.

Reply:
Well, i haven't used Maven but i see some other use cases in which this feature will be useful. Let's say that you want to add a date or other values only present during the first release run as part of the description, but you want the original date or values to be preserved. 

Will try to setup some "test before build" behavior before merging this PR. 

Thanks for the explanation, i didn't get it the first time.😃